kolibri/utils/cli.py

```diff
@@ -491,3 +491,11 @@ def _get_env_vars():
 @configure.command(help="List all available environment variables to configure Kolibri")
 def list_env():
     click.echo_via_pager(_get_env_vars())
+
+
+@configure.command(cls=KolibriDjangoCommand, help="Setup Kolibri")
+def setup():
+    """
+    Setup Kolibri.
+    """
+    logger.info("Kolibri has successfully been setup")
```

Comment:
Looking at the source of `KolibriDjangoCommand`, it's not clear how or why this works:

https://github.com/learningequality/kolibri/blob/366b49c42652a248fcf0057c32a810bac4a8d392/kolibri/utils/cli.py#L169-L194

If `KolibriDjangoCommand` is implicitly initializing home folders in the background, this information should be added to the docstring.

Reply:
And reading it closer, it says "calls the initialize function"

🤦